tools/diff-processor/detector/detector.go

```diff
@@ -33,6 +37,16 @@ type Field struct {
 	Tested bool
 }
 
+type MissingDocDetails struct {
+	FilePath string
+	Fields   []MissingDocField
+}
+
+type MissingDocField struct {
+	Field   string
+	Section string
```

Comment:
I assume section indicates if this field is in argument or attribute section, IIRC? Can you add a small comment for it?

Reply:
Section is removed since not differentiate argument and attribute now.